src/main/java/com/gitrekt/resort/controller/BrowseRoomsScreenController.java

```diff
@@ -42,6 +42,9 @@
 
     @FXML
```

Comment:
This file should not have been included in this commit, but we will fix that later - I'll still accept the PR assuming everything else checks out. The problem is that you branched off of your existing branch, rather than off of dev, meaning both branches are included in this pr.